libs/orchestrator-ui-components/src/lib/components/Subscription/ProcessesTimeline.tsx

```diff
@@ -0,0 +1,109 @@
+    EuiSpacer,
+    EuiLoadingContent,
+} from '@elastic/eui';
+
+import { ProcessStatusBadge } from '../Badge/ProcessStatusBadge';
+import { SubscriptionProcess, useSubscriptionProcesses } from '../../hooks';
+
+const Card = (processInfo: SubscriptionProcess) => {
+    return (
+        <div style={{ marginTop: 5 }}>
+            <table width="100%" bgcolor={'#F1F5F9'}>
+                <tr>
+                    <td
+                        valign={'top'}
+                        style={{
+                            width: 250,
+                            padding: 10,
+                            borderBottom: 'solid 1px #ddd',
+                        }}
+                    >
+                        <b>ID</b>
+                    </td>
+                    <td style={{ padding: 10, borderBottom: 'solid 1px #ddd' }}>
+                        <a
+                            href={`https://orchestrator.dev.automation.surf.net/processes/${processInfo.process.pid}`}
+                        >
+                            {processInfo.pid}
+                        </a>
+                    </td>
+                </tr>
+                <tr>
+                    <td
+                        valign={'top'}
+                        style={{ padding: 10, borderBottom: 'solid 1px #ddd' }}
+                    >
+                        <b>Status</b>
+                    </td>
+                    <td style={{ padding: 10, borderBottom: 'solid 1px #ddd' }}>
+                        <ProcessStatusBadge
+                            processStatus={processInfo.process.last_status}
+                        />
+                    </td>
+                </tr>
+                <tr>
+                    <td
+                        valign={'top'}
+                        style={{ padding: 10, borderBottom: 'solid 1px #ddd' }}
+                    >
+                        <b>Started at</b>
+                    </td>
+                    <td style={{ padding: 10, borderBottom: 'solid 1px #ddd' }}>
+                        2-11-2021 13:47:43 - Duration: 00:00:024
+                    </td>
+                </tr>
+                <tr>
+                    <td valign={'top'} style={{ padding: 10 }}>
+                        <b>Started by</b>
+                    </td>
+                    <td style={{ padding: 10 }}>Hans Trompert</td>
+                </tr>
+            </table>
+        </div>
+    );
+};
+
+const RenderProcess = (processInfo: SubscriptionProcess) => {
+    return (
+        <EuiComment
+            username={
+                processInfo.workflow_target ? processInfo.workflow_target : ''
+            }
+            timelineAvatarAriaLabel={processInfo.process.workflow}
+            timelineAvatar={<EuiAvatar name="C" />}
+        >
+            {Card(processInfo)}
+        </EuiComment>
+    );
+};
+
+export type ProcessesTimelineProps = {
+    subscriptionId: string;
+};
+
+export const ProcessesTimeline: FC<ProcessesTimelineProps> = ({
```

Comment:
`ProcessesTimeline` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.

Reply:
#96